components/clp-py-utils/clp_py_utils/telemetry.py

```
@@ -0,0 +1,54 @@
+    if disable_env_var in TELEMETRY_DISABLE_VALUES or dnt_env_var in TELEMETRY_DISABLE_VALUES:
+        metrics.set_meter_provider(NoOpMeterProvider())
+        logger.debug("OpenTelemetry metrics disabled.")
+        return
+
+    try:
+        exporter = OTLPMetricExporter()
+        reader = PeriodicExportingMetricReader(exporter)
+        provider = MeterProvider(metric_readers=[reader])
+        metrics.set_meter_provider(provider)
+        logger.debug("OpenTelemetry metrics initialized successfully.")
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)